rear_camera_py/src/calculator.py

```diff
@@ -0,0 +1,72 @@
+"""計算を実行するモジュール(ブロックエリア情報取得用).
+
+@author: kawanoichi
+"""
+
+import numpy as np
+
+
+class Calculator:
```

Comment:
```suggestion
class LineCalculator:
```
とか？

Lineクラスにして、コンストラクタに複数座標を渡してインスタンス化、line.calc_distance_coord()みたいな感じで距離算出とかでもいいかも